tests/test_floors/test_update_floor.py

```diff
@@ -42,3 +43,28 @@ def test_update_floor_that_is_not_in_admin_location_throws_errors(self):
             self,
             update_floor_mutation,
             "You are not authorized to make changes in Kampala")
+
+    def test_database_connection_error(self):
+        """
+        test a user friendly message is returned to a user when database
+        cannot be reached
+        """
+        BaseTestCase().tearDown()
+        CommonTestCases.admin_token_assert_in(
+            self,
+            update_floor_mutation,
+            "The database cannot be reached"
+            )
+
+    def test_update_floors_without_floors_relation(self):
+        """
+        Testing for floor creation without floor relation
```

Comment:
Are we testing for creating floor or updating floor? And is it possible to do the same without floor relation?

Reply:
I've rectified the docstring error... It's meant to be update not creation. rectified 
@Bonifase